src/userLayout/Checkout.tsx

```diff
@@ -27,16 +33,20 @@ const PayBooking = () => {
         console.log(error);
```

Comment:
We can add a toast message here to tell the user what is going wrong. In the current implementation, the user will click on the submit button but will not get any response back if something is going wrong.